### ISSUE TITLE
`makeDropPane` fails for base64 encoded drops

### ISSUE THREAD

**Issue by IanVS** [filepicker/filepicker-js#52]
When attempting to drag and drop an image from a google image search, I get a `400 BAD REQUEST` from `https://www.filepicker.io/api/store/S3?key=xxx&plugin=js_lib&_cacheBust=xxx&plugin=js_lib` with a response of:

```
Invalid URL file data:image/jpeg;base64,/9j/4AAQSkZJRgABAQ... - invalid schema
```

**Comment by sjmrt** [comment]
Hi Ian, do you have a place we could replicate this and a specific file that this might be occurring with?

**Comment by IanVS** [comment]
Yep, you can use the example on https://www.filestack.com/docs/html-widget/drag-and-drop-pane

Then, search google images for "cat": https://www.google.com/search?tbm=isch&q=cat&tbs=imgo:1 and drag the first image into the example drag-and-drop-pane.

You should see: 
![image](https://cloud.githubusercontent.com/assets/4616705/21063690/009f0a3a-be25-11e6-9f76-68c6a0688a96.png)

Thanks for the quick response!

**Comment by IanVS** [comment]
Hi @sjmrt, were you able to confirm that this is unintended behavior?
